Fix misleading wording about quotation

vfaronov opened the pull request:
RFC 7231 Section 5.3.2 requires quoting even a single URI
in a media range parameter, because it contains characters that 
are not allowed in an unquoted token (RFC 7230 Section 3.2.6).

gabesullice commented:
Thanks @vfaronov, good catch! Would you mind updating this PR to the tip of gh-pages?

vfaronov committed:
Fix misleading wording about quotation

RFC 7231 Section 5.3.2 requires quoting even a single URI
in a media range parameter, because it contains characters that
are not allowed in an unquoted token (RFC 7230 Section 3.2.6).

vfaronov commented:
@gabesullice Done.

gabesullice commented:
Thank you!
